cirq/sim/density_matrix_simulator_test.py

```diff
@@ -807,22 +823,45 @@ def test_density_matrix_trial_result_repr():
                 "qubit_map={cirq.LineQubit(0): 0}))""")
 
 
+class XAsOp(cirq.Operation):
+
+    def __init__(self, q):
+        # coverage: ignore
```

Comment:
If the test class method is not covered, just `raise NotImplementedError()` instead of bothering to implement it.

If the test class method is covered, no need for the coverage ignore.

Reply:
Done